azext_iot/digitaltwins/commands_twins.py

```diff
@@ -44,7 +44,12 @@ def update_twin(cmd, name_or_hostname, twin_id, json_patch, resource_group_name=
 
 def delete_twin(cmd, name_or_hostname, twin_id, resource_group_name=None, etag=None):
     twin_provider = TwinProvider(cmd=cmd, name=name_or_hostname, rg=resource_group_name)
-    return twin_provider.delete(twin_id, etag=etag)
+    return twin_provider.delete(twin_id=twin_id, etag=etag)
+
+
+def delete_all_twin(cmd, name_or_hostname, resource_group_name=None, etag=None):
```

Comment:
I think usage of etag here is awkward since each twin and relationship has its own etag value. The lower level delete operation would fail on the first mismatched etag.